templates/register.html

```diff
@@ -0,0 +1,34 @@
+{{ define "content" }}
+
+<main class="auth-main">
+  <div class="auth-main__content">
+    <img class="auth-main__image" src="dist/images/google-go.jpg" alt="Workflow">
+
+    <h2 class="auth-main__title">
+      Register a new account
+    </h2>
+    
+    {{ if .error }}
+      <div class="alert--danger">
```

Comment:
This class is NOT BEM and it breaks (no styles):

![image](https://user-images.githubusercontent.com/696529/100057075-56504400-2e59-11eb-8a8f-f6a7cff4215e.png)


Instead, it should be:


```suggestion
      <div class="alert alert--danger">
```

Each class has a purpose:

- `.alert`: shared styles for all alerts e.g. size, dimensions, positioning
- `.alert--danger`: styles only for error messages e.g. color, background-color

Reply:
it was my sloppy.
Fixed in a6bb092cd493d30437f722b3f56d908fefa8dfd8

Reply:
Still having some issues with the latest code. While the class names are now correctly added, the styles still do not show:

![image](https://user-images.githubusercontent.com/696529/100192008-ba3f3f00-2f23-11eb-8e4a-34829a4affc4.png)